0-ad-history-mod-package/simulation/templates/units/han/champion_chariot_academy.xml

```diff
@@ -13,6 +13,12 @@
     <Requirements>
       <Techs datatype="tokens">-unlock_champion_cavalry phase_city</Techs>
     </Requirements>
+    <History>
+       The use of chariots in ancient China peaked between the 8th and 5th centuries BC. They were usually drawn by two or four horses and carried three men: the charioteer who steered the chariot, an archer who stood typically on the charioteer's left, and a soldier carrying a dagger-axe to the charioteer's right. 
+       The dagger-axe or “ge” had a dagger-like blade on a long handle. Due to its length, the dagger-axe could even reach enemy chariots. The chariot's crew wore various types of armor, like leather, and the horses too.
+       Chariots were used for attack, defense, and pursuit, and there were different types for the different needs. Because chariots were very vulnerable when surrounded, they were also generally accompanied by infantry. When chariots were used to attack, they fought as shock units, charging at the enemy line at their full speed of around 13 mph (20 km/h). The enemy's  formations oftentimes could not withstand the impact and were broken up and usually ended with defeat. In fact, for some time, a state's military power was mostly determined by the number of chariots it could deploy.
```

Comment:
```suggestion
       Chariots were used for attack, defense, and pursuit, and there were different models for the different needs. Because chariots were vulnerable when surrounded, they were generally accompanied by infantry. When chariots were used to attack, they fought as shock units, charging at the enemy line at their full speed of around 13 mph (20 km/h). The enemy's  formations oftentimes could not withstand the impact and were broken up, this usually resulted in a defeat. In fact, for some time, a state's military power was mostly determined by the number of chariots it could deploy.
```